src/components/AddBook.jsx

```diff
@@ -33,7 +32,7 @@ const AddBook = () => {
   return (
     <section className="addBook">
       <h2>ADD NEW BOOK</h2>
-      <form className="addBookForm">
+      <form>
         <input
           value={values.title || ''}
```

Comment:
- [ ] For your `ADD NEW BOOK` section, I notice you have a lot of bolder/dark colors for the title, form element border, and form element place holders texts, This is a bit different from the original design. Please take a look at it and address the color contrast.   
### Your design
![image](https://github.com/Cisco-the-wayword/Bookstore/assets/69077061/b1095b8d-d054-4ac3-8d4e-fdb77e04a1b7)

### Original design 
![image](https://github.com/Cisco-the-wayword/Bookstore/assets/69077061/720b047a-440b-4271-817a-6e8ce95f6cd6)